liota/dccs/iotcc.py

```diff
@@ -71,7 +71,6 @@ def __init__(self, con):
         thread.start()
         # Wait for Subscription to be complete and then proceed to publish message
         time.sleep(0.5)
```

Comment:
There must be a better way to do this. Sleeping is never the right choice. Why wouldn't we just block waiting for the reply? If the server is slow what ensures 500ms is enough, anyway?

Reply:
Will raise a separate bug for this.